aws-cpp-sdk-core-tests/utils/DateTimeTest.cpp

```diff
@@ -63,7 +63,7 @@ TEST(DateTimeTest, TestRFC822Parsing_WrongFormat)
 
 TEST(DateTimeTest, TestISO_8601Parsing)
 {
-    const char* gmtDateStr = "2002-10-02T08:05:09Z";
+    const char* gmtDateStr = "2002-10-02T08:05:09.000Z";
```

Comment:
We don't actually want to send this format. We don't have millisecond precision in ISO Dates and it isn't actually used by any of the other SDKs.